cypress/e2e/ModelizerPage/AllView/settings/git.settings.feature

```diff
@@ -0,0 +1,151 @@
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+    Then I expect "positive" toast to appear with text "We have access to your repository 🥳!"
+    And I expect "[data-cy=\"git-form\"]" is closed
+    And I expect localstorage field "projects" is '{"{{ projectName }}":{"id":"{{ projectName }}","git":{"repository":"https://github.com/ditrit/leto-modelizer-project-test","username":"test","token":"test"}}}'
+
+  Scenario: Set bad repository url should display an error
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-repository-input\"]" text "bad"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+    Then I expect "[data-cy=\"git-form\"] [role=\"alert\"]" is "Invalid repository url"
+
+  Scenario: Set empty repository should display an error
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+    Then I expect field "[data-cy=\"git-form\"] [data-cy=\"git-repository-input\"]" is ""
+    And I expect "[data-cy=\"git-form\"] [role=\"alert\"]" is "Please type something"
+
+  Scenario: Set empty username should display an error
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-repository-input\"]" text "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+    Then I expect field "[data-cy=\"git-form\"] [data-cy=\"git-username-input\"]" is ""
+    And I expect "[data-cy=\"git-form\"] [role=\"alert\"]" is "Please type something"
+
+  Scenario: Set empty token should display an error
+    When I click on "[data-cy=\"project-settings\"]"
+    And I click on "[data-cy=\"git-settings-menu\"]"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-repository-input\"]" text "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set on "[data-cy=\"git-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And I click on "[data-cy=\"git-form\"] [data-cy=\"git-form-submit\"]"
+    Then I expect field "[data-cy=\"git-form\"] [data-cy=\"git-token-input\"]" is ""
+    And I expect "[data-cy=\"git-form\"] [role=\"alert\"]" is "Please type something"
+
+  Scenario: Each project should have its own value of git configuration
```

Comment:
I don't understand the purpose of this test.

Reply:
If I save configuration, it won't update all other configuration, check that configuration is related to a project

Reply:
I just deeply check and I understand that your test :
* create a new project
* set la git config
* back to homepage
* create a new project
-> the new project has no git config
So this test should check if the config of the first project on local storage is (still) existing to be accurate.
(could be the set of git config which didn't work...)